autogen.sh

```diff
@@ -24,7 +24,8 @@ cd -               # back to original directory
 
 if [ -z "$NO_CONFIGURE" ]
 then
-    $srcdir/configure --enable-maintainer-mode "$@"  ||  exit $?
+    # prepend 'sh -x' if you want to debug
+    "$srcdir"/configure --enable-maintainer-mode "$@" 2>&1 | tee configure.log || exit $?
```

Comment:
```suggestion
    "$srcdir"/configure --enable-maintainer-mode "$@" || exit $?
```

The `|| exit $?` has no effect since I added a pipe-tee to a log file, so remove that.